The Iterable and Mapping were removed from collections in Python 3.10
Import Iterable and Mapping from collections.abc
Author: Cintia Willemyns (Vrije Universiteit Brussel)
diff -Naru evcouplings-0.1.1.orig/evcouplings/align/protocol.py evcouplings-0.1.1/evcouplings/align/protocol.py
--- evcouplings-0.1.1.orig/evcouplings/align/protocol.py	2024-06-04 17:15:24.409905000 +0200
+++ evcouplings-0.1.1/evcouplings/align/protocol.py	2024-06-04 17:16:54.492901448 +0200
@@ -8,7 +8,8 @@
 
 """
 
-from collections import OrderedDict, Iterable
+from collections import OrderedDict
+from collections.abc import Iterable
 import re
 from shutil import copy
 import os
diff -Naru evcouplings-0.1.1.orig/evcouplings/compare/pdb.py evcouplings-0.1.1/evcouplings/compare/pdb.py
--- evcouplings-0.1.1.orig/evcouplings/compare/pdb.py	2024-06-04 17:15:24.419850000 +0200
+++ evcouplings-0.1.1/evcouplings/compare/pdb.py	2024-06-04 17:52:31.882593767 +0200
@@ -5,7 +5,8 @@
   Thomas A. Hopf
 """
 
-from collections import OrderedDict, Iterable
+from collections import OrderedDict
+from collections.abc import Iterable
 from os import path
 from urllib.error import HTTPError
 
diff -Naru evcouplings-0.1.1.orig/evcouplings/couplings/mapping.py evcouplings-0.1.1/evcouplings/couplings/mapping.py
--- evcouplings-0.1.1.orig/evcouplings/couplings/mapping.py	2024-06-04 17:15:24.405238000 +0200
+++ evcouplings-0.1.1/evcouplings/couplings/mapping.py	2024-06-04 17:52:59.165312780 +0200
@@ -7,7 +7,7 @@
   Anna G. Green (MultiSegmentCouplingsModel)
 """
 
-from collections import Iterable
+from collections.abc import Iterable
 from copy import deepcopy
 from evcouplings.couplings.model import CouplingsModel
 import pandas as pd
diff -Naru evcouplings-0.1.1.orig/evcouplings/couplings/model.py evcouplings-0.1.1/evcouplings/couplings/model.py
--- evcouplings-0.1.1.orig/evcouplings/couplings/model.py	2024-06-04 17:15:24.407628000 +0200
+++ evcouplings-0.1.1/evcouplings/couplings/model.py	2024-06-04 17:53:16.476317130 +0200
@@ -6,7 +6,7 @@
 Authors:
   Thomas A. Hopf
 """
-from collections import Iterable
+from collections.abc import Iterable
 from copy import deepcopy
 
 from numba import jit
diff -Naru evcouplings-0.1.1.orig/evcouplings/utils/app.py evcouplings-0.1.1/evcouplings/utils/app.py
--- evcouplings-0.1.1.orig/evcouplings/utils/app.py	2024-06-04 17:15:24.424719000 +0200
+++ evcouplings-0.1.1/evcouplings/utils/app.py	2024-06-04 17:53:56.971793813 +0200
@@ -14,7 +14,7 @@
 import re
 from copy import deepcopy
 from os import path, environ
-from collections import Mapping
+from collections.abc import Mapping
 
 import click
 
diff -Naru evcouplings-0.1.1.orig/evcouplings/utils/tracker/mongodb.py evcouplings-0.1.1/evcouplings/utils/tracker/mongodb.py
--- evcouplings-0.1.1.orig/evcouplings/utils/tracker/mongodb.py	2024-06-04 17:15:24.421959000 +0200
+++ evcouplings-0.1.1/evcouplings/utils/tracker/mongodb.py	2024-06-04 17:54:25.411050098 +0200
@@ -16,7 +16,7 @@
 
 import os
 from datetime import datetime
-from collections import Mapping
+from collections.abc import Mapping
 
 from pymongo import MongoClient, errors
 import gridfs
